plugin/src/test/java/org/opennms/plugins/cloud/config/CertificateImporterTest.java

```diff
@@ -92,7 +93,7 @@ public void shouldImportCertificates() throws Exception {
     assertTrue(value.startsWith("-----BEGIN PRIVATE KEY-----"));
     value = credentials.getAttribute(Type.tokenvalue.name());
     assertNotNull(value);
-    assertTrue(value.startsWith(TOKEN_KEY_ACME));
+    assertTrue(value.startsWith("acme"));
```

Comment:
why not using org.opennms.plugins.cloud.config.SecureCredentialsVaultUtil.TOKEN_KEY ?

Reply:
org.opennms.plugins.cloud.config.SecureCredentialsVaultUtil.TOKEN_KEY resolves to "token".
In the zip file it has "acme"